schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+create database vet_clinic;
+CREATE TABLE animals(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	name VARCHAR(50),
+  	date_of_birth DATE,
+  	escape_attempts INT,
+  	neutered BOOLEAN,
+  	weight_kg FLOAT
```

Comment:
# Optional
- [ ] It would be better if we change the FLOAT data type to `decimal` so we have more flexibility of the float figures

Reply:
@addod19 Thank you for your suggestion 🙏